internal/service/searchindex/data_source_search_indexes.go

```diff
@@ -131,7 +131,13 @@ func flattenSearchIndexes(searchIndexes []admin.SearchIndexResponse, projectID,
 			}
 			searchIndexesMap[i]["fields"] = fieldsMarshaled
 		}
-	}
 
+		storedSource := searchIndexes[i].LatestDefinition.GetStoredSource()
+		strStoredSource, errStoredSource := MarshalStoredSource(storedSource)
+		if errStoredSource != nil {
```

Comment:
would be great to have plural data source read leverage the logic defined in singular data source for populating each index. Can be a small refactor ticket.

Reply:
thx, created CLOUDP-261461